frontend/src/game/PhaserGame.tsx

```diff
@@ -109,22 +108,46 @@ export const PhaserGame = forwardRef<IRefPhaserGame, IProps>(function PhaserGame
   if (calculateVariableValue(gameData.ups.acquired, GameVariable.ButtonAvailable) === 1) {
     visible = true;
   }
-
-  return (
-    <div style={{ flex: 1, textAlign: "center", width: "100%", height: "100%" }}>
-      <button
-        style={{
-          visibility: visible ? "visible" : "hidden",
-          marginBottom: "1%",
-          width: "15%",
-          height: "5%",
-          fontSize: "18px"
-        }}
-        onClick={() => EventBus.emit("collect-all")}
-      >
-        Collect All
-      </button>
-      <div id="game-container" style={{ flex: 1, textAlign: "center", width: "100%", height: "90%" }}></div>
-    </div>
-  );
+  function reduceCooldown() {
+    setCooldown(cooldown - 1);
+    dispatch(setPreviousCooldown(cooldown));
+  }
+  if (cooldown === 0) {
+    return (
```

Comment:
CSS is evil but if you could modify the button so it doesn't wrap like this that'd be great

![image](https://github.com/user-attachments/assets/69870ab3-007b-4fc1-9863-4b76555d9442)